keyboards/xelus/rs60/rev2/chconf.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2022 QMK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * This file was auto-generated by:
+ *    `qmk chibios-confmigrate -i keyboards/acheron/austin/chconf.h -r platforms/chibios/common/configs/chconf.h`
+ */
+
+#pragma once
```

Comment:
Is this file needed, or is it just copied and pasted?

Reply:
I've copy pasted, since i've looked at the Mode/m75s files which uses this. Unsure if this is required.

Reply:
gonna merge for now.

Though, the reason I ask, is that the blackpill boards also seem to have this config file, but don't need it. In fact, it was a bit more stable/faster after removing it.